test/support/conn_case.ex

```diff
@@ -74,6 +74,19 @@ defmodule ScreenplayWeb.ConnCase do
 
           {conn, user}
 
+        tags[:api_unauthenticated] ->
+          conn =
+            Phoenix.ConnTest.build_conn() |> Plug.Conn.put_req_header("x-api-key", "1234")
+
+          {conn, nil}
+
+        tags[:api_authenticated] ->
+          conn =
+            Phoenix.ConnTest.build_conn()
+            |> Plug.Conn.put_req_header("x-api-key", "local_api_key")
```

Comment:
This won't work if someone changes their API key from the default one in `.envrc.template` — we should fetch the real one that's configured. `put_req_header("x-api-key", Application.fetch_env!(...))`

Reply:
Ahh, I did think about that, but was a little too optimistic about the local value not changing from the template.